src/kernels/level3/xgemm_part1.opencl

```diff
@@ -138,6 +138,25 @@ R"(
   #endif
 #endif
 
+#if USE_SUBGROUP_SHUFFLING == 1 && SUBGROUP_SHUFFLING_GCN == 1
+  #define SUBGROUP_SIZE 32              // Assumes subgroup size is always 4 on AMD GCN GPUs
```

Comment:
On the left you write 32, on the right you write 4, one of them probably is incorrect?

Reply:
Yes. It should be 32 (for Navi cards). Will change the comment.